src/main/java/com/mparticle/kits/ApptimizeKit.java

```diff
@@ -0,0 +1,135 @@
+package com.mparticle.kits;
+
+import android.content.Context;
+
+import com.apptimize.Apptimize;
+import com.mparticle.MPEvent;
+import com.mparticle.MParticle;
+import com.mparticle.commerce.CommerceEvent;
+
+import java.math.BigDecimal;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class ApptimizeKit extends KitIntegration implements KitIntegration.AttributeListener, KitIntegration.EventListener {
+
+    public static final String APP_KEY = "appKey";
+    public static final String ALIAS_KEY = "mparticleAlias";
+    public static final String CUSTOMER_ID_KEY = "mparticleCustomerId";
+    public static final String VIEWED_EVENT_FORMAT = "Viewed %s Screen";
+
+    @Override
+    protected List<ReportingMessage> onKitCreate(Map<String, String> settings, Context context) {
+        String appKey = getSettings().get(APP_KEY);
```

Comment:
I used getSettings, getContext in analogy to existing flurry/localytics implementations, but it's kind of weird since there were a settings/context passed in?